docs/pages/reference/backends.mdx

```diff
@@ -620,6 +620,182 @@ Config option priority is applied in the following order:
   are not supported in certain regions or environments or are only supported in GovCloud.
 </Admonition>
 
+## Athena
+
+If you are running Teleport on AWS, you can use
+[Athena](https://aws.amazon.com/athena/) based audit log system that uses
+[Parquet files](https://parquet.apache.org/) stored on
+[S3](https://aws.amazon.com/s3/) as a storage back-end to achieve High
+Availability. Athena backend supports only one type of Teleport data:
+
+- Audit log events
+
+The athena audit logs backend provides better scalability and searchability in
```

Comment:
```suggestion
The Athena audit logs back-end provides better scalability and searchability in
```
- Capitalize proper nouns
- Use "back-end" for consistency with the rest of this page